interpreter/terminal_interface/start_terminal_interface.py

```diff
@@ -630,7 +631,7 @@ def start_terminal_interface(interpreter):
 
     # Set attributes on interpreter
     for argument_name, argument_value in vars(args).items():
-        if argument_value != None:
+        if argument_value is not None:
```

Comment:
?

Reply:
https://peps.python.org/pep-0008/#programming-recommendations

"Comparisons to singletons like None should always be done with is or is not, never the equality operators."